libs/light/neopixel.ts

```diff
@@ -1153,7 +1126,7 @@ namespace light {
         public showFrame(strip: NeoPixelStrip): void { }
     }
 
-    class RainbowCycleAnimation extends NeoPixelAnimation {
+    export class RainbowCycleAnimation extends NeoPixelAnimation {
```

Comment:
I think you can just use: `export const Rainbow: NeoPixelAnimation = new RainbowCycleAnimation(...)` without `export` here. But maybe it's good to be able to derive them.